src/jvmTest/kotlin/wizard/GeneratedProjectTest.kt

```diff
@@ -144,6 +144,20 @@ class GeneratedProjectTest {
             dependencies = allDependencies
         )
         val dir = projectInfo.writeToDir(workingDir)
+        val deviceId = checkCommand(
+            dir = dir,
+            command = listOf(
+                "xcrun",
+                "simctl",
+                "list",
+                "devices",
+                "available"
```

Comment:
it is better to use `xcrun simctl list devices available -j` and parse a JSON output

Reply:
JSON output is applied.